Source/DafnyCore/Compilers/GoLang/Compiler-go.cs

```diff
@@ -202,11 +197,32 @@ private struct Import {
 
       importWriter.WriteLine("{0} \"{1}\"", id, path);
 
-      if (!import.SuppressDummy) {
-        if (id == "os") {
-          importDummyWriter.WriteLine("var _ = os.Args");
+      bool isType;
+      string memberName = null;
+      if (id == "os") {
+        memberName = "Args";
+        isType = false;
+      } else {
+        isType = true;
+        if (import.ExternModule != null) {
+          var attributes = Attributes.Find(import.ExternModule.Attributes, "dummyImportMember");
```

Comment:
Definitely glad we're not documenting this officially, but it might be worth adding a note on this attribute to https://github.com/dafny-lang/dafny/blob/master/Source/DafnyStandardLibraries/CONTRIBUTING.md, just so anyone that might need to use it is at least aware of how it works.